src/web/components/MostViewed/MostViewed.tsx

```diff
@@ -98,7 +98,7 @@ function buildSectionUrl(sectionName?: string) {
     const sectionsWithoutPopular = ['info', 'global'];
     const hasSection =
         sectionName && !sectionsWithoutPopular.includes(sectionName);
-    const endpoint = `/c${hasSection ? `/${sectionName}` : ''}.json`;
+    const endpoint = `/most-read${hasSection ? `/${sectionName}` : ''}.json`;
```

Comment:
is this a related change?

Reply:
It's fixing a typo from another change but because this PR was built off the Storybook PR, and because it fails silently, I decided to fix it in both places